src/pipes/passwords-match.pipe.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { PasswordsMatchPipe } from './passwords-match.pipe';
+import { ArgumentMetadata, BadRequestException } from '@nestjs/common';
+import { CREATE_USER_DTO_TEST_OBJ, CREATE_USER_DTO_TEST_OBJ_WITH_UNMATCHING_PASSWORDS } from '../../test/test.constants';
+
+describe('PasswordsMatchPipe', () => {
+  let passwordsMatchPipe: PasswordsMatchPipe;
+  let metaData: ArgumentMetadata;
+
+  beforeEach(() => {
+    passwordsMatchPipe = new PasswordsMatchPipe();
+  });
+
+  /* Tests that when password and passwordConfirmation match,
+   the same CreateUserDto obj that is passed to the pipeline, is returned */
+  describe('Test Matching Passwords', () => {
+    it('should return the same CreateUserDto', () => {
+      expect(passwordsMatchPipe).toBeDefined();
```

Comment:
I don't think there needs to be a check in every test for the pipe to be defined. Maybe just create one test that checks.